THIRD-PARTY-NOTICES.TXT

```diff
@@ -1206,3 +1206,29 @@ As an exception, if, as a result of your compiling your source code, portions
 of this Software are embedded into a machine-executable object form of such
 source code, you may redistribute such embedded portions in such object form
 without including the above copyright and permission notices.
+
+
+License for MurmurHash3 (https://github.com/aappleby/smhasher)
```

Comment:
We do not need the whole smhasher test suite. We can just reference the specific file that we need and that is under a different license:

https://github.com/aappleby/smhasher/blob/master/src/MurmurHash3.cpp#L2-L3

Reply:
Thanks, updated, let me know if it needs better wording